c_src/image/image.h

```diff
@@ -0,0 +1,34 @@
+/*
+#  Created by Boyd Multerer on 2021-03-28
+#  Copyright © 2021 Kry10 Limited. All rights reserved.
+#
+*/
+
+#pragma once
+
+#include "scenic_types.h"
+#include "tommyhashlin.h"
+
+typedef struct {
+  sid_t id;
+  int32_t image_id;
+  uint32_t width;
+  uint32_t height;
+  uint32_t format;
+  void* p_pixels;
+  tommy_hashlin_node  node;
+} image_t;
+
+typedef enum {
+  image_format_file = 0,
+  image_format_gray = 1,
+  image_format_gray_alpha = 2,
+  image_format_rgb = 3,
+  image_format_rgba = 4,
+  image_format_size = 0xffffffff, // ensure enum is 32 bits wide
```

Comment:
Minor preference for `ALL_CAPS` or `kEnumValue`, but not worth holding up a PR over. :)

Reply:
This enum and others that I added I changed to be `ALL_CAPS` for the enum values